spec/controllers/api/venues_controller_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::VenuesController, type: :controller do
+  let(:organizer) { create(:user, role: :organizer) }
+
+  describe 'GET #show' do
+    let(:venue_attributes) { %w[id address description] }
+    before do
+      @header = organizer.create_new_auth_token
+      request.headers.merge!(@header)
+    end
+
+    context 'when valid' do
+      let(:venue) { create(:venue) }
+
+      it 'returns json response with venue' do
+        get :show, params: { id: venue.id }
+        expect(json_response.keys).to include('id', 'address', 'description', 'people_capacity')
+      end
+    end
+  end
+
+  describe 'POST #create' do
```

Comment:
please check that number of records changed/not changed on creation/destroying
please use to change and not_to change matchers

Reply:
done